test/gurka/test_golf_cart_preference.cc

```diff
@@ -0,0 +1,80 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+#if !defined(VALHALLA_SOURCE_DIR)
+#define VALHALLA_SOURCE_DIR
+#endif
+
+using namespace valhalla;
+
+// Check that golf carts prefer lower classes of roads
+class GolfCartPreference : public ::testing::Test {
+protected:
+  static gurka::map map;
+
+  static void SetUpTestSuite() {
+    constexpr double gridsize = 100;
+
+    // A network with a mixture of road classes, all of which are legally traversable
+    const std::string ascii_map = R"(
```

Comment:
Combine this into the access tests to reduce the number of executables that need to be built.